### PR TITLE
Fix: China Battlemaster Paradrop delivers to wrong location

### LINKED ISSUE #1779: Tank Paradrop Delivers Wrong Location to Tanks
Tank Paradrop is using same ```PreOpenDistance``` with infantry paradrop without ```ParachuteDirectly``` (guided parachutes). Because of that, they are parachuting further back. ```ParachuteDirectly = Yes``` causing sloppy situation. It's the best not using it.

### Original:

https://user-images.githubusercontent.com/103841664/226494768-472213ae-84e4-42c7-b712-670ec6f5510e.mp4

### Fixed:

https://user-images.githubusercontent.com/103841664/226494860-ad0bcfff-2d27-4909-b461-7b979345b5fc.mp4

### PR DESCRIPTION
**Pull request by MavroGada2**
* Fixes #1779 

Tanks now paradrop to decal. ```PreOpenDistance``` changed to 100 from 300.

### CODE REVIEW

**Comment by xezon** [comment]
Needs yaml file for change log.

File: Patch104pZH/Design/Changes/v1.0/1780_tank_paradrop_power.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+title: Fixes China Battlemaster Paradrop delivers to wrong location
+
+changes:
+  - fix: Fixes China Tank Paradrop delivers to wrong location. It no longer paradrops behind the decal.
+
+labels:
+  - china
+  - minor
+  - bug
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/1780_tank_paradrop_power.yaml]: Please add the v1.0 label too. And you can sort the labels by alphabet.

Comment (MavroGada2) [reply Patch104pZH/Design/Changes/v1.0/1780_tank_paradrop_power.yaml]: I thought I couldn't do it so I closed the tab.

**Review by xezon — APPROVED** [review]
Looks good.

**Pull request merged**